### PR TITLE
Staging deploy

### PR DESCRIPTION
**Pull request by tiagojsag**
- bump ruby version (not really necessary, but hey, it helps)
- run npm install, cp env file and run npm run build on capistrano deploy
- a few more things that are needed to keep capistrano happy

### CODE REVIEW

**Comment by agnessa** [comment]
I tried running the specs locally and the build fails on ruby 2.4.2, passes on 2.4.1. I think you haven't changed the version of ruby for travis, which is why CI still passes.

**Comment by tiagojsag** [comment]
Yeah, I had fixed it, just on another branch. Now it's here, and I also updated travis

**Comment by agnessa** [comment]
great, thanks!

File: config/deploy.rb [hunk 1]
```diff
@@ -32,9 +32,27 @@
...
-set :rvm_ruby_version, '2.4.1'
\ No newline at end of file
+namespace :npm do
+  after 'npm:install', 'npm:build'
+
+  task :build do
+    on roles fetch(:npm_roles) do
+      within fetch(:npm_target_path, release_path) do
+        with fetch(:npm_env_variables, {}) do
+          upload! "frontend/.env.#{fetch(:stage)}", "#{current_path}/frontend/.env"
```
Comment (agnessa) [inline config/deploy.rb]: does this upload a local `.env` file? could we not do it following the same principle as the backend .env file, which is kept on it's target environment between releases?

File: config/deploy.rb [hunk 2]
```diff
@@ -32,9 +32,27 @@
...
+
 namespace :deploy do
   after :finishing, 'deploy:cleanup'
   after 'deploy:publishing', 'deploy:restart'
 end
 
-set :rvm_ruby_version, '2.4.1'
\ No newline at end of file
+namespace :npm do
+  after 'npm:install', 'npm:build'
```
Comment (agnessa) [inline config/deploy.rb]: what is the full deployment command now?

File: config/deploy.rb (hunk 1 above)
Comment (tiagojsag) [reply config/deploy.rb]: :+1:

File: config/deploy.rb (hunk 2 above)
Comment (tiagojsag) [reply config/deploy.rb]: same as before, cap <env> deploy.

**Comment by tiagojsag** [comment]
@agnessa ready for review

**Pull request merged**